tests/Integration/Behaviour/Features/Context/Domain/Product/DeleteProductFeatureContext.php

```diff
@@ -43,34 +41,19 @@ class DeleteProductFeatureContext extends AbstractProductFeatureContext
      *
      * @param string $reference
      */
-    public function deleteProduct(string $reference): void
+    public function deleteProductForDefaultShop(string $reference): void
     {
-        try {
-            $this->getCommandBus()->handle(new DeleteProductCommand(
-                $this->getSharedStorage()->get($reference)
-            ));
-        } catch (ProductException $e) {
-            $this->setLastException($e);
-        }
+        $this->deleteProduct($reference, ShopConstraint::shop($this->getDefaultShopId()));
     }
 
     /**
      * @When I bulk delete following products:
      *
      * @param TableNode $productsList
      */
-    public function bulkDeleteProducts(TableNode $productsList): void
+    public function bulkDeleteProductsForDefaultShop(TableNode $productsList): void
```

Comment:
Same just a suggestion
```suggestion
    public function bulkDeleteProductsFromDefaultShop(TableNode $productsList): void
```